homeassistant/components/alexa/resources.py

```diff
@@ -216,11 +216,14 @@ def serialize_labels(resources):
         labels = []
         for label in resources:
             if label in AlexaGlobalCatalog.__dict__.values():
-                label = {"@type": "asset", "value": {"assetId": label}}
+                label = [{"@type": "asset", "value": {"assetId": label}}]
             else:
-                label = {"@type": "text", "value": {"text": label, "locale": "en-US"}}
+                label = [
+                    {"@type": "text", "value": {"text": label, "locale": "en-GB"}},
```

Comment:
We need a way to support capability resources for multiple languages, and in addition limit capabilities to allowed countries... I guess this could work for now for all English.

Reply:
Found this while reviewing the Alexa Documentation:

[The locale in which the string is localized. Currently, the only supported value is en-US.](https://developer.amazon.com/en-US/docs/alexa/device-apis/resources-and-assets.html#label-object)

Reply:
Interesting. I’ll retest this one again, it caused me some confusion initially - the Alexa simulator (forced en-US) worked fine but my Alexa devices (en-GB at the time, now en-US) did not.